app/src/main/java/com/codingblocks/cbonlineapp/commons/SheetAdapter.kt

```diff
@@ -28,6 +29,16 @@ class SheetAdapter(val items: ArrayList<SheetItem>) : BaseAdapter() {
                 DrawableCompat.setTintMode(drawable, PorterDuff.Mode.SRC_IN)
                 setCompoundDrawablesRelativeWithIntrinsicBounds(drawable, null, null, null)
             }
+        } else {
+            view.textView.apply {
+                text = sheetItem.name
+                setTextColor(resources.getColor(R.color.black))
+                var drawable = context.getDrawable(sheetItem.icon)!!
+                drawable = DrawableCompat.wrap(drawable)
+                DrawableCompat.setTint(drawable, resources.getColor(R.color.black))
+                DrawableCompat.setTintMode(drawable, PorterDuff.Mode.SRC_IN)
+                setCompoundDrawablesRelativeWithIntrinsicBounds(drawable, null, null, null)
+            }
```

Comment:
Earlier view was being recycled. So previously selected item view was also being highlighted. So moved in else block so that even if recycled, it will be non highlighted